libs/blocks/card/cardUtils.js

```diff
@@ -36,10 +36,17 @@ export const addFooter = (links, container, merch) => {
   });
 };
 
-export const addWrapper = (el, section, cardType) => {
+export const addWrapper = (el, cardType) => {
+  const fragment = el.closest('.fragment');
+  if (fragment) {
+    const fragmentSection = el.closest('.section');
+    fragmentSection?.replaceWith(el);
+    fragment.style.display = 'flex';
```

Comment:
Could this be achieved via a CSS modifier class? Inline styles are a bit more difficult to maintain in the long run and it makes sense to have styles grouped together

Reply:
Any chance this can still be moved to a modifier class instead of an inline style?